<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-13T17:31:50 IST" name="Default suite" finished-at="2022-04-13T17:32:02 IST" duration-ms="12511">
    <groups>
    </groups>
    <test started-at="2022-04-13T17:31:50 IST" name="Default test" finished-at="2022-04-13T17:32:02 IST" duration-ms="12511">
      <class name="com.simplilearn.demo.Assertion">
        <test-method signature="launch()[pri:0, instance:com.simplilearn.demo.Assertion@6c80d78a]" started-at="2022-04-13T17:31:50 IST" name="launch" finished-at="2022-04-13T17:31:57 IST" duration-ms="7187" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method signature="TryFacebook()[pri:0, instance:com.simplilearn.demo.Assertion@6c80d78a]" started-at="2022-04-13T17:31:57 IST" name="TryFacebook" finished-at="2022-04-13T17:32:01 IST" depends-on-methods="com.simplilearn.demo.Assertion.launch" duration-ms="4675" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TryFacebook -->
        <test-method signature="login()[pri:0, instance:com.simplilearn.demo.Assertion@6c80d78a]" started-at="2022-04-13T17:32:02 IST" name="login" finished-at="2022-04-13T17:32:02 IST" depends-on-methods="com.simplilearn.demo.Assertion.TryFacebook" duration-ms="532" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [Facebook – log in or sign up] but found [Facebook – log in or sign upqwer]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Facebook – log in or sign up] but found [Facebook – log in or sign upqwer]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at com.simplilearn.demo.Assertion.login(Assertion.java:54)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
      </class> <!-- com.simplilearn.demo.Assertion -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
